headless/src/jsMain/kotlin/dev/fritz2/headless/components/combobox.kt

```diff
@@ -798,6 +801,16 @@ class Combobox<E : HTMLElement, T>(tag: Tag<E>, id: String?) : Tag<E> by tag, Op
 
             attr(Aria.activedescendant, activeIndexStore.data.map { it?.let(::itemId) })
 
+            // This manual computation is needed due to a bug in floating-ui preventing us from using the built-in
+            // auto-update functionality.
+            // See https://github.com/floating-ui/floating-ui/issues/1740
+            // TODO: Remove once the floating-ui auto-update bug is resolved
```

Comment:
I would prefer `FIXME` over `TODO`

Also I propose to start with the `FIXME: This manual...` and then add all of your comment. This would group the whole comment, which will become obsolete, once `floating-ui` fixes that bug:

```kotlin
            // FIXME: This manual computation is needed due to a bug in floating-ui preventing us from using the built-in
            //  auto-update functionality.
            //  See https://github.com/floating-ui/floating-ui/issues/1740
            //  Remove once the floating-ui auto-update bug is resolved
```
(You might wanna change the line breaks - but beware the extra spaces that group the comment lines)